ClimsoftVer4/ClimsoftVer4/ucrMetadataStationElement.vb

```diff
@@ -5,9 +5,9 @@
             'SetUpTableEntry 
             SetUpTableEntry("stationelement")
 
-            ucrDataLinkElementID.SetTableNameAndField("stationelement", "describedBy")
+            ucrDataLinkElementID.SetTableNameAndField("obselement", "elementId")
             ucrDataLinkElementID.PopulateControl()
-            ucrDataLinkElementID.SetDisplayAndValueMember("describedBy")
+            ucrDataLinkElementID.SetDisplayAndValueMember("elementId")
```

Comment:
@Wycklife  this is not necessary. We an element selector which does all this for you internally